src/ucp/core/ucp_worker.c

```diff
@@ -2729,8 +2732,11 @@ ucp_worker_do_keepalive_progress(ucp_worker_h worker)
         return 0;
     }
 
+    ucs_trace("worker %p: keepalive round", worker);
```

Comment:
This calls in a loop! Maybe do log once per 100 or 1000 iterations, or decrease log level? Logs now full of it:
![image](https://user-images.githubusercontent.com/1121987/109674030-41353080-7b7f-11eb-97ca-37cdc3816ea3.png) and not much useful information